src/seedu/addressbook/ui/Formatter.java

```diff
@@ -34,6 +35,17 @@ public String format(String... messages) {
         return sb.toString();
     }
 
+    /** Returns a concatenated string of each object
+     *
+     */
+    public static String getPrintableString(Printable... printables) {
```

Comment:
What would be a better place to have this method?

Reply:
Should I implement this method at ReadOnlyPerson.java instead?